TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/FileTypeIconViewHelper.php

```diff
@@ -18,16 +18,16 @@
 use TYPO3\Media\Domain\Service\FileTypeIconService;
 
 /**
- * Renders an <img> HTML tag for a filetype icon for a given TYPO3.Media's asset instance
+ * Renders an <img> HTML tag for a filetype icon for a given Neos.Media's asset instance
  *
  * = Examples =
  *
  * <code title="Rendering an asset filetype icon">
- * <typo3.media:fileTypeIcon asset="{assetObject}" alt="a filetype icon" height="16" />
+ * <neos.media:fileTypeIcon asset="{assetObject}" alt="a filetype icon" height="16" />
```

Comment:
Even though this is just a comment (used to render documentation!), it will lead people to the use of a not (yet) existing ViewHelper. So, please only merge with the actual vendor name change.